### DISCUSSION TITLE
size of file choices

### DISCUSSION THREAD

**Discussion by catsandme** [aclap-dev/video-downloadhelper#809] [Help and Q&A]
### Your question

I updated video downloadhelper today.  It has a totally different look now.  I can't figure out how to change the size of the video I want to download.  Is that still possible? 

### Addon & CoApp version and details

I updated video downloadhelper today.  It has a totally different look now.  I can't figure out how to change the size of the video I want to download.  Is that still possible?

**Comment by GrampaWildWilly** [comment]
![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/258373c2-8d27-4da2-a5ab-673b6a9b3558)
